scripts/test/runtests.ps1

```diff
@@ -44,7 +49,7 @@ pushd "$TestBinRoot"
 
 # Run each test project
 $TestProjects | ForEach-Object {
-    & "corerun.exe"  "xunit.console.netcore.exe" "$_.dll" -xml "$_.xml" -notrait category=failing
+    & "corerun.exe"  "xunit.console.netcore.exe" "$_.dll" -xml "$_.xml" -notrait category=failing -parallel none
```

Comment:
If our tests are written poorly I agree it'll happen. I'd revert this change as its unrelated to your changes.